base/nullable.jl

```diff
@@ -61,6 +87,26 @@ end
 
 get(x::Nullable) = x.isnull ? throw(NullException()) : x.value
 
+
+"""
+    isnull(x::Nullable) -> Bool
+
+Is the [`Nullable`](:obj:`Nullable`) object `x` null, i.e. missing a value?
```

Comment:
I think this would be better as a statement instead of a question.

> Returns if the ...

or something